src/test/resources/pages/instructorFeedbackResultsEditCommentByDifferentInstructor.html

```diff
@@ -995,7 +995,7 @@ <h4 class="modal-title">
                                     </table>
                                   </div>
                                   <div class="form-group">
-                                    <div class="panel panel-default panel-body content-editor" id="responsecommenttext-1-0-1-1-1" spellcheck="false" style="position: relative;">
+                                    <div class="panel panel-default panel-body content-editor empty" id="responsecommenttext-1-0-1-1-1" spellcheck="false" style="position: relative;">
```

Comment:
Why is this change necessary?

Reply:
I don't know why this file got changed in God Mode. Reverting it.

Reply:
It's happening quite often and is an unrelated GodMode change. I am investigating it.

I am guessing the test is unstable and hence fails and corrects the HTML files at times (which shouldn't happen)